src/features/DecisionTreeGenerator/Forms/FormTreeCardMutation/useFormTreeCardMutation.tsx

```diff
@@ -200,6 +209,18 @@ const useFormTreeCardMutation = () => {
     defaultValues,
     modalOpen,
   ]);
+  const isEditModal = modalOpen === "edit";
+
+  useEffect(() => {
```

Comment:
Ici ça serait bien d'ajouter un petit commentaire pour qu'on sache à quoi sert cette effect.

On peu aussi clean le code : 

```suggestion
    if (isUnique) {
      setUniqueNameErrorMessage("");
      return;
    }

    setUniqueNameErrorMessage(t("mustBeUnique", { ns: "form" }));
```